configuration-tool/tinymce-configuration-assistant.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+<body>
+  <div id="configurator">
+  </div>
+  <script src="https://cdn.tiny.cloud/1/qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc/tinymce/5/tinymce.min.js" referrerpolicy="origin"></script>
+  <script src="../js/configurator/configurator.js" referrerpolicy="origin"></script>
```

Comment:
Why is this getting embedded on the docs site? That is a really bad idea and this should be hosted by actual infrastructure similar to the skin tool.